orders/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from .views import CartView
+
+urlpatterns = [
+    path('/cart/add', CartView.as_view())
```

Comment:
```suggestion
    path('/cart', CartView.as_view())
```
add라는 말이 post method로 대체되야 할것 같네요